test/unit/gemm/warp/gemm_mixed_input_sm80.cu

```diff
@@ -120,6 +120,251 @@ TEST(SM80_warp_gemm_mixed_input_tensor_op_crosswise_i8_f16, 128x128x64_64x64x64_
 }
 
 
+TEST(SM80_warp_gemm_mixed_input_tensor_op_crosswise_i8_f16, 64x64x64_64x64x64_16x8x16) {
```

Comment:
this test name repeats 8 times.

Reply:
Thanks for catching it. I fixed it. I was thinking how it got repeated, and it is probably during rebase against each of my local commits and me accepting the changes.